glitch/repr/inter.py

```diff
@@ -70,16 +70,25 @@ def __init__(self, name: str, value: str, has_variable: bool) -> None:
         self.name: str = name
         self.value: str = value
         self.has_variable: bool = has_variable
+        self.variables: list = []
 
     def __repr__(self) -> str:
         value = repr(self.value).split('\n')[0]
-        name = self.name.split('.')[-1]
-        return f"{name}:{value}"
+        name = self.name
+        if value == "None":
+            variables = self.variables
```

Comment:
I would use `self.variables` in the return instead of defining the local variable.

Reply:
I Agree, Thank you!